test/e2e/multi_nic_test.go

```diff
@@ -36,7 +36,7 @@ const (
 	additionalSubnetVarKey = "NUTANIX_ADDITIONAL_SUBNET_NAME"
 )
 
-var _ = Describe("Nutanix Subnets [PR-Blocking]", func() {
+var _ = Describe("Nutanix Subnets", func() {
```

Comment:
Add a label to run all the CAPX specific tests

Reply:
added